rest_framework/fields.py

```diff
@@ -880,7 +885,10 @@ def to_internal_value(self, data):
             self.fail('invalid')
 
     def to_representation(self, value):
-        return float(value)
+        value = float(value)
+        if self.localize:
+            value = localize_input(value)
```

Comment:
Localizing the output doesn't make sense here. FloatField should represent as floats, not as a localized string.